packages/compass/scripts/check-bundled-app-size.ts

```diff
@@ -0,0 +1,42 @@
+import { promises as fs } from 'fs';
+import path from 'path';
+
+const MAX_SIZE_MB = 200;
+const MIN_SIZE_MB = 50;
```

Comment:
non blocking: do we need to check for the minimum size? It should be safe to say that if we miss big pieces from the bundle the e2e tests would fail